### ISSUE TITLE
esbuild dependency version

### ISSUE THREAD

**Issue by adz** [phenax/esbuild-plugin-elm#5]
On install getting a warning when using with esbuild 0.12.24:

```
warning " > esbuild-plugin-elm@0.0.4" has incorrect peer dependency "esbuild@^0.8.51".
```

No issues in running at all, so wondering if this can be bumped up to latest?

**Comment by phenax** [comment]
Weird. The caret `^` is supposed to handle all minor version updates as it only describes "compatible with".

~~I am unable to reproduce this, can you show me the relevant parts of your package json?~~

Edit: Found a way to reproduce the issue. Working on a fix.

**Comment by adz** [comment]
Oh cool. Thanks!
I'm not too familiar with js packaging.

**Comment by phenax** [comment]
This has been patched in version 0.0.5
